lib/rules/lint-no-curly-component-invocation.js

```diff
@@ -0,0 +1,98 @@
+const Rule = require('./base');
+const { transformTagName } = require('../helpers/curly-component-invocation');
+const IN_BUILT_HELPERS = [
+  'action',
```

Comment:
I pulled this list from https://api.emberjs.com/ember/release/classes/Ember.Templates.helpers